src/utils/cancel.tsx

```diff
@@ -0,0 +1,5 @@
+import { History } from 'history';
+
+export function handleCancel (namespace: string, policyType:string ,data , history: History)  {
+  history.push(`/k8s/all-namespaces/kuadrant.io~v1alpha1~TLSPolicy`)
```

Comment:
```suggestion
  history.push(`/k8s/all-namespaces/kuadrant.io~v1alpha1~TLSPolicy`)
```

maybe something like

```suggestion
  history.push(`/k8s/all-namespaces/kuadrant.io~v1alpha1~${policyType}`)
```

guess we'd need an apiVersion too